providers/aws/components/baseline/state.ftl

```diff
@@ -2,6 +2,39 @@
 [#assign LOCAL_SSH_PRIVATE_KEY_RESOURCE_TYPE = "sshPrivKey" ]
 [#assign AWS_SSH_KEY_PAIR_RESOURCE_TYPE = "sshKeyPair" ]
 
+[#function formatSegmentCMKId ]
```

Comment:
with the work I've done on services, I think these functions should be in the id file not the state file. hat was the logic for moving them?

Reply:
To get rid of the KMS id file mainly. These functions where for baseline only Id formatting and are for legacy support